themes/digital.gov/layouts/shortcodes/img.html

```diff
@@ -48,7 +48,7 @@
       src="{{- if ne $prefix $thisuid -}}
         {{- $thisuid -}}
       {{- else -}}
-        {{- $imgBaseCDN -}}_w800.{{- $imgExt -}}
+        {{- $imgBaseCDN -}}_w1200.{{- $imgExt -}}
```

Comment:
This fixes the sizing issue on desktop. I tested on several recent pages and those mentioned by @ToniBonittoGSA